packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseServersArray-test.js

```diff
@@ -0,0 +1,74 @@
+const { Fury } = require('fury');
+const { expect } = require('../../chai');
+const parse = require('../../../../lib/parser/oas/parseServersArray');
+const Context = require('../../../../lib/context');
+
+const { minim: namespace } = new Fury();
+
+describe('Servers Array', () => {
+  let context;
+
+  beforeEach(() => {
+    context = new Context(namespace);
+  });
+
+  it('warns when it is not an array', () => {
+    const servers = new namespace.elements.Object();
+
+    const parseResult = parse(context, servers);
+
+    expect(parseResult.length).to.equal(1);
+    expect(parseResult).to.contain.warning("'Servers Array' is not an array");
+  });
+
+  it('parses correctly when there is a single server', () => {
+    const server = new namespace.elements.Array([
+      {
+        url: 'https://user.server.com/1.0',
+        description: 'The production API server',
+      },
+    ]);
+
+    const parseResult = parse(context, server);
+
+    expect(parseResult.length).to.equal(1);
+
+    const servers = parseResult.get(0);
```

Comment:
```suggestion
    const hosts = parseResult.get(0);
```

Perhaps we can call this `hosts` to match API Elements terminology as the `parse` method returns API Elements.

Reply:
word replaced for all the cases that are the result of the parser